lib/modules/manager/custom/jsonata/schema.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from 'zod';
+
+const DepObjectSchema = z.object({
+  currentValue: z.string().optional(),
```

Comment:
```suggestion
  currentValue: z.string(),
```
Shouldn't be `currentValue` be required based on the above?

Reply:
It doesn't need to be there, as we can also configure a `currentValueTemplate`

Reply:
What is the use case for `currentValueTemplate`?

Reply:
In regex manager's context it is currently being used to provide the template for modifying the captured curretValue or to directly provide one.

Reply:
It is possible/reasonable to make `currentValue`, `depName` and `datasource` manadatory for the jsonata manager..because if user wants a single value then its possible to do so directly using the query unlike regex manager.

For example:

_If we want a constant datasource_
```
query = packages.{"depName": depNameVar, "datasource": "npm"}
```
Same for `depName` & `currentValue`. And we can also modify the "depNameVar" using the functionality provided by jsonata queries.